packages/node/src/handlers.ts

```diff
@@ -304,6 +305,11 @@ export function errorHandler(options?: {
           }
         }
 
+        _scope.addEventProcessor(event => {
+          addExceptionMechanism(event, { type: 'middleware', handled: false });
```

Comment:
Idk what the type here should be. I was going to write "express", but this middleware can be used by many different libraries, unlike the sveltkit and angular integrations.

Reply:
I think middleware is fine for this, makes sense to me!